docs/architecture/rpc.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The RPC protocol
+authors: "Tim McMackin"
+last_update:
+  date: 6 November 2023
+---
+
+The Tezos RPC (Remote Procedure Call) protocol is a specification for a REST API that clients use to interact with Tezos nodes and nodes use to communicate with each other.
+Clients use this protocol to submit transactions and get information about the state of the blockchain, such as account balances and contract storage.
+Tezos nodes act as servers and accept HTTP requests from clients and other nodes via this protocol.
+
+Tezos RPC uses JSON to send and receive data, but it does not adhere to the JSON-RPC specification.
```

Comment:
Maybe worth adding the list of calls: https://tezos.gitlab.io/shell/rpc.html#rpc-index-shell

Reply:
Good suggestion. Added that link.